modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/base_controller.rb

```diff
@@ -42,7 +42,21 @@ def shared_form_validation(form_number)
           validate_json_schema(form_number.upcase)
         end
 
-        def submit_power_of_attorney(poa_code, form_number)
+        def validate_registration_number!
+          rn = form_attributes.dig('serviceOrganization', 'registrationNumber')
```

Comment:
I'm assuming the 2122a signatures ticket will update this to work for the `representative` object?